templates/build-and-test-job.yml

```diff
@@ -110,13 +110,16 @@ jobs:
           POWERAPPS_SPECFLOW_BINDINGS_TEST_ADMIN_PASSWORD2: $(Extra Admin User Password)
           POWERAPPS_SPECFLOW_BINDINGS_TEST_URL: ${{ parameters.environmentUrl }}
       - task: SonarCloudAnalyze@1
+        enabled: false
```

Comment:
I think SonarCloud and WhiteSource Bolt should be enabled again?

Reply:
Thanks Max....your suggested has been applied